docs/api/url_reference/sectionid_criterion.md

```diff
@@ -0,0 +1,14 @@
+# SectionId Criterion
+
+The [`SectionId` URL Criterion](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/API/Repository/Values/URL/Query/Criterion/SectionId.php)
+matches URLs based on the ID of the related content Section.
+
+## Arguments
+
+- `sectionIds` - array of ints representing the IDs of the related content Sections
```

Comment:
```suggestion
- `sectionIds` - int(s0 representing the IDs of the related content Section(s)
```
to be consistent with the other pages

Reply:
In this case it has to be array of ints otherwise you get an error